Code/GraphMol/NewCIPLabelling/Descriptor.hpp

```diff
@@ -0,0 +1,97 @@
+//
+//
+//  Copyright (C) 2020 Greg Landrum and T5 Informatics GmbH
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#pragma once
+
+namespace RDKit {
+namespace NewCIPLabelling {
+
+static const std::string CIP_LABEL_KEY = "cip.label";
+static const std::string CONF_INDEX = "conf.index";
+
+/**
+ * Defines a descriptor which can be assigned to an atom to indicate the type of
+ * chirality (if there is any). Each descriptor defines it's general @{link
+ * Type} which can be useful when comparing centres of different geometry.
+ *
+ */
+enum class Descriptor {
+  NONE,  // Unspecified
+  UNKNOWN,
+  ns,  // Other
+  /**
+   * Tetrahedral
+   */
+  R,
+  S,
+  r,
+  s,
+  /**
+   * Cis/Trans
+   */
+  seqTrans,
+  seqCis,
+  E,
+  Z,
+  /* Axial */
+  M,
+  P,
+  m,
+  p,
+
+  SP_4,
+  TBPY_5,
+  OC_6
+};
+
+static std::string to_string(const Descriptor& desc) {
```

Comment:
`operator<<` ?

Reply:
I had that, but replaced it with `to_string()' in the last commit, it felt more handy, and I didn't see the point in having both.